plugins/modules/dynamodb_table.py

```diff
@@ -206,6 +213,11 @@
     returned: success
     type: str
     sample: ACTIVE
+table_class:
+    description: The current table class.
+    returned: when state present
+    type: str
+    sample: STANDARD_INFREQUENT_ACCESS
```

Comment:
This is the actual RETURN statement, I guess it should reflect this.
```
changed: [testhost] => {
    "billing_mode": "PROVISIONED",
    "changed": true,
    "hash_key_name": "id",
    "hash_key_type": "NUMBER",
    "indexes": [],
    "invocation": {},
    "range_key_name": null,
    "range_key_type": null,
    "read_capacity": 1,
    "region": "us-east-1",
    "resource_actions": [
        "dynamodb:ListTagsOfResource",
        "dynamodb:DescribeTable",
        "dynamodb:CreateTable"
    ],
    "table": {
        "arn": "arn:aws:dynamodb:us-east-1:721066863947:table/ansible-test-table",
        "attribute_definitions": [
            {
                "attribute_name": "id",
                "attribute_type": "N"
            }
        ],
        "billing_mode": "PROVISIONED",
        "creation_date_time": "2022-02-04T13:36:01.578000+00:00",
        "id": "533b45fe-0870-4b66-9b00-d2afcfe96f19",
        "item_count": 0,
        "key_schema": [
            {
                "attribute_name": "id",
                "key_type": "HASH"
            }
        ],
        "name": "ansible-test-14482047-alinas-mbp",
        "provisioned_throughput": {
            "number_of_decreases_today": 0,
            "read_capacity_units": 1,
            "write_capacity_units": 1
        },
        "size": 0,
        "status": "ACTIVE",
        "table_arn": "arn:aws:dynamodb:us-east-1:721066863947:table/ansible-test-table",
        "table_id": "533b45fe-0870-4b66-9b00-d2afcfe96f19",
        "table_name": "ansible-test-table",
        "table_size_bytes": 0,
        "table_status": "ACTIVE",
        "tags": {}
    },
    "table_class": null,
    "table_name": "ansible-test-table",
    "table_status": "ACTIVE",
    "tags": {},
    "write_capacity": 1
}

```

Reply:
> This is the actual RETURN statement, I guess it should reflect this.
> 
> ```
> changed: [testhost] => {
>     "billing_mode": "PROVISIONED",
>     "changed": true,
>     "hash_key_name": "id",
>     "hash_key_type": "NUMBER",
>     "indexes": [],
>     "invocation": {},
>     "range_key_name": null,
>     "range_key_type": null,
>     "read_capacity": 1,
>     "region": "us-east-1",
>     "resource_actions": [
>         "dynamodb:ListTagsOfResource",
>         "dynamodb:DescribeTable",
>         "dynamodb:CreateTable"
>     ],
>     "table": {
>         "arn": "arn:aws:dynamodb:us-east-1:721066863947:table/ansible-test-table",
>         "attribute_definitions": [
>             {
>                 "attribute_name": "id",
>                 "attribute_type": "N"
>             }
>         ],
>         "billing_mode": "PROVISIONED",
>         "creation_date_time": "2022-02-04T13:36:01.578000+00:00",
>         "id": "533b45fe-0870-4b66-9b00-d2afcfe96f19",
>         "item_count": 0,
>         "key_schema": [
>             {
>                 "attribute_name": "id",
>                 "key_type": "HASH"
>             }
>         ],
>         "name": "ansible-test-14482047-alinas-mbp",
>         "provisioned_throughput": {
>             "number_of_decreases_today": 0,
>             "read_capacity_units": 1,
>             "write_capacity_units": 1
>         },
>         "size": 0,
>         "status": "ACTIVE",
>         "table_arn": "arn:aws:dynamodb:us-east-1:721066863947:table/ansible-test-table",
>         "table_id": "533b45fe-0870-4b66-9b00-d2afcfe96f19",
>         "table_name": "ansible-test-table",
>         "table_size_bytes": 0,
>         "table_status": "ACTIVE",
>         "tags": {}
>     },
>     "table_class": null,
>     "table_name": "ansible-test-table",
>     "table_status": "ACTIVE",
>     "tags": {},
>     "write_capacity": 1
> }
> ```

@alinabuzachis I am wondering if everything bar the actual `table` object and maybe `table_status` should be deprecated and future dropped as the `table` object contains all actual fields from the AWS describe call whereas the rest is basically a mirror of the inputted params.

Reply:
Maybe if I doc just `table` for now ?

Reply:
You can do something like this in RETURN:

```
...
table:
    description: Add description
    type: complex
    sample: {
        "arn": "arn:aws:dynamodb:us-east-1:721066863947:table/ansible-test-table",
        "attribute_definitions": [
            {
                "attribute_name": "id",
                "attribute_type": "N"
            }
        ],
        "billing_mode": "PROVISIONED",
        "creation_date_time": "2022-02-04T13:36:01.578000+00:00",
        "id": "533b45fe-0870-4b66-9b00-d2afcfe96f19",
        "item_count": 0,
        "key_schema": [
            {
                "attribute_name": "id",
                "key_type": "HASH"
            }
        ],
        "name": "ansible-test-14482047-alinas-mbp",
        "provisioned_throughput": {
            "number_of_decreases_today": 0,
            "read_capacity_units": 1,
            "write_capacity_units": 1
        },
        "size": 0,
        "status": "ACTIVE",
        "table_arn": "arn:aws:dynamodb:us-east-1:721066863947:table/ansible-test-table",
        "table_id": "533b45fe-0870-4b66-9b00-d2afcfe96f19",
        "table_name": "ansible-test-table",
        "table_size_bytes": 0,
        "table_status": "ACTIVE",
        "tags": {}
    }
....
```